lib/modules/showImages.js

```diff
@@ -1114,8 +1114,8 @@ addModule('showImages', (module, moduleID) => {
 			image.classList.add('RESImage');
 			image.id = `RESImage-${RESUtils.randomHash()}`;
 			image.src = sourceImage.src;
-			image.style.maxWidth = `${module.options.maxWidth.value}px`;
```

Comment:
While we're in here, let's upgrade this to sanitize and validate:

```
const maxWidth = parseInt(module.options.maxWidth.values, 10);
if (maxWidth > 0) image.style.maxWidth = `${maxWidth}px`;
```

and same for maxHeight